simtel_baseline.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+
+
+def fill_baseline_r0(event_stream, n_bins0=5, n_bins1=10, method='data'):
+
+    for count, event in enumerate(event_stream):
+
+        for telescope_id in event.r0.tels_with_data:
+
+            if method == 'data':
```

Comment:
this "if method .. else other_method" could also be implemented as two independend functions.

Reply:
yes lets_make to functions and the the main one does it the `sim_tel` way and the `data` way